roles/dhis2/meta/main.yml

```diff
@@ -56,6 +56,7 @@ dependencies: # noqa 701
 - role: collectd
   collectd_deployed_app_name: "dhis2"
   collectd_deployed_app_version: "{{ dhis_version }}"
+  collectd_version: "{{ server_collectd_version }}"
```

Comment:
Should the value not be from `dhis_collectd_version` instead? It seems to be going against how variables are named in the role.

Reply:
updated